internal/servers/console_proxy_core_test.go

```
@@ -0,0 +1,272 @@
+	"net"
+	"sync"
+	"time"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+
+	"github.com/osac-project/fulfillment-service/internal/console"
+)
+
+// pipeRWC wraps a net.Conn as io.ReadWriteCloser for testing.
+type pipeRWC struct {
```

golangci-lint (2.12.2)

[error] type pipeRWC is unused

(unused)